lib/lotus/commands/runner.rb

```diff
@@ -0,0 +1,53 @@
+require 'lotus/environment'
+require 'lotus/application'
+
+module Lotus
+  module Commands
+    # Evaluate a file or expression with the preloaded environment and application.
+    #
+    # It is run with:
+    #
+    #   `bundle exec lotus runner ./my_script.rb`
+    #
+    # or
+    #
+    #   `bundle exec lotus runner 'puts Lotus.env'`
+    #
+    # @since x.x.x
+    # @api private
+    class Runner
+      # @since x.x.x
+      # @api private
+      class NoArgumentGivenException < ::StandardError
+      end
+
+      # @param options [Hash] Environment's options
+      # @param expression_or_file [String] Expression or file to evaluate
+      #
+      # @since x.x.x
+      # @see Lotus::Environment#initialize
+      # @see Lotus::Environment#require_application_environment
+      # @see Lotus::Application.preload!
+      def initialize(options, expression_or_file)
+        @expression_or_file = expression_or_file
+        raise NoArgumentGivenException.new if @expression_or_file.nil?
```

Comment:
@AlfonsoUceda What about `ArgumentError`? It's a common Ruby pattern for this kind of situations :smile: